src/shapes/Text.js

```diff
@@ -468,6 +468,8 @@ define([
 
             if (dc.pickingMode) {
                 this.pickColor = dc.uniquePickColor();
+                program.loadColor(gl, this.pickColor);
+                program.loadOpacity(gl, 1);
             }
 
             // Compute the effective visibility. Use the current value if picking.
```

Comment:
This isn't part of the fix, but there could be an `else` here instead of the second if. The comment is also detached from part of the logic, which is above.